routes/web.php

```diff
@@ -9,9 +15,67 @@
+    Route::get('/roles/{id}/edit', [RoleController::class, 'edit'])->name('roles.edit');
+    Route::put('/roles/{id}', [RoleController::class, 'update'])->name('roles.update');
+    Route::delete('/roles/{id}', [RoleController::class, 'destroy'])->name('roles.destroy');
+// ---------- // Role // ------------
+
+// ---------- Benutzer -----------------
+    Route::get('/users', [User2Controller::class, 'index'])->name('users.index');
+    Route::get('/users/create', [User2Controller::class, 'create'])->name('users.create');
+    Route::post('/users', [User2Controller::class, 'store'])->name('users.store');
+    Route::get('/users/{id}', [User2Controller::class, 'show'])->name('users.show');
+    Route::get('/users/{id}/edit', [User2Controller::class, 'edit'])->name('users.edit');
+    Route::put('/users/{id}', [User2Controller::class, 'update'])->name('users.update');
+    Route::delete('/users/{id}', [User2Controller::class, 'destroy'])->name('users.destroy');
+// ---------- // Benutzer // ------------
+
+    Route::get('/tools', [Tool2Controller::class, 'index'])->name('tools.index');
+    Route::get('/tools/create', [Tool2Controller::class, 'create'])->name('tools.create');
+    Route::post('/tools', [Tool2Controller::class, 'store'])->name('tools.store');
+    Route::get('/tools/{id}', [Tool2Controller::class, 'show'])->name('tools.show');
+    Route::get('/tools/{id}/edit', [Tool2Controller::class, 'edit'])->name('tools.edit');
+    Route::put('/tools/{id}', [Tool2Controller::class, 'update'])->name('tools.update');
+    Route::delete('/tools/{id}', [Tool2Controller::class, 'destroy'])->name('tools.destroy');
+
+// --------------Statistik -----------------
+    Route::get('/statistics', [StatisticController::class, 'index'])->name('statistics.index');
+// ---------- // Statistik // --------------
+
+// ---------- send Emails ------------------
+    Route::get('/emails/create', [Email2Controller::class, 'create'])->name('email.form');
+    Route::post('/emails', [Email2Controller::class, 'store'])->name('send.email');
+// ---------- // send Emails // ------------
+
+// ---------- Maintenance Mode ------------------
+    Route::get('/toggle-maintenance-mode', [MaintenanceModeController::class, 'index'])->name('maintenance.mode');
+    Route::post('/toggle-maintenance-mode', [MaintenanceModeController::class, 'toggleMaintenanceMode'])->name('maintenance.mode');
+// ---------- //Maintenance Mode// --------------
+});
+
+Route::get('/dashboard', function () {
+    return view('dashboard');
+})->middleware(['auth', 'verified'])->name('dashboard');
+
+Route::middleware('auth')->group(function () {
+    Route::get('/profile', [ProfileController::class, 'edit'])->name('profile.edit');
+    Route::patch('/profile', [ProfileController::class, 'update'])->name('profile.update');
+    Route::delete('/profile', [ProfileController::class, 'destroy'])->name('profile.destroy');
```

Comment:
I had a default route for Laravel authentication, but I have deleted it because I don't want to use it anymore. I have added custom authentication instead.